forge-game/src/main/java/forge/game/card/CounterEnumType.java

```diff
@@ -371,6 +371,8 @@ public enum CounterEnumType {
 
     STUN("STUN", 226, 192, 165),
 
+    TAKEOVER("TAKEOVER", 63, 49, 191),
```

Comment:
I think the rule is string in "" should be 5 or less characters... TOVER? TAKEO?